.prettierrc

```diff
@@ -0,0 +1,3 @@
+semi: true
```

Comment:
@archae0pteryx Is there a reason you're overriding defaults here?

Reply:
I was trying to match the code that was already written as much as i could. Semi's seem to be the pattern. I was trying to limit the number of lint errors. I have no opinion either way, as long as everyone agrees on them.

Reply:
Yeah. Prettier ships with defaults and they've been applied to all the existing files recently. So it makes sense that running prettier again would make no changes. 

My preference is to leave it with Prettier's defaults unless someone strongly argues that it's not the  "react" way.

Reply:
totally. i will reset that.